docs/modules/mobile/cloud-deploy.md

```diff
@@ -0,0 +1,75 @@
+<%(TOC)%>
+
+# Connect to a cloud deployment
+
+<%(Callout message="
+Before reading this document, make sure you have read:
+
+* [Setting up Android Support for the GDK]({{urlRoot}}/modules/mobile/setup-android)
+* [Setting up iOS Support for the GDK]({{urlRoot}}/modules/mobile/setup-ios)
+* [Development authentication flow](https://docs.improbable.io/reference/latest/shared/auth/development-authentication)
```

Comment:
So small nit - but this doc has a huge amount of information that you don't need as we abstract it away. Is there a glossary entry that's a bit more concise that we can link to?

Reply:
would like to, but we don't have one? can link to the new locator flow